docs/mvc/security/cors-policy/sample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNet.Cors.Core;
+using Microsoft.AspNet.Mvc;
+
+namespace CorsMvc.Controllers
+{
+    [EnableCors("AllowSpecificOrigin")]
+    public class HomeController : Controller
+    {
+        [EnableCors("AllowSpecificOrigin")] 
```

Comment:
Specifying the same policy at controller and action is not useful, probably the example can use a different policy here and an action with no policy.